### DISCUSSION TITLE
Enhancement: Plugin Add

### DISCUSSION THREAD

**Discussion by theanubhavk** [danny-avila/LibreChat#853] [Feature Requests & Suggestions]
Nothing just wanna say that ,it will be too much enjoyable if more plugin was added .

**Comment by danny-avila** [comment]
I will add them as they are introduced via PR if they work well. I myself am adding a few in #845 

You can always feel free to add [ChatGPT Plugins](https://docs.librechat.ai/features/plugins/chatgpt_plugins_openapi.html#adding-a-plugin) or even [make your own LangChain plugin](https://docs.librechat.ai/features/plugins/make_your_own.html)
